website/docs/intro/upgrading.mdx

```diff
@@ -69,6 +69,8 @@ If you suspect the failure may be the result of a bug in OpenTofu,
 Now you are ready to migrate. Run `tofu init` in the directory where your code resides. OpenTofu will download
 any providers and modules referenced in your configuration from the OpenTofu registry.
 
+**Note: If you are using the S3 backend** - You would need to run `tofu init -reconfigure` to reinitialize the backend.
```

Comment:
Wondering if this part needs to be more explicit and be its own step

Reply:
`You would` -> `You will`

I think this is fine, but it could be in a :::note block